devops/src/database_manager.py

```
@@ -0,0 +1,53 @@
+class DatabaseManager:
+    def __init__(self):
+        self.rds = boto.rds
+        self.sgm = security_group_manager
+
+    def create_database(self):
+        database_sg = self.sgm.create_database_security_group()
+
+        username = os.getenv('DB_USERNAME')
+        password = os.getenv('DB_PASSWORD')
+        if username is None or password is None:
+            raise Exception('Db username or password not found in env variables')
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)